### DISCUSSION TITLE
fail-learn-reset

### DISCUSSION THREAD

**Discussion by giscus** [WorkSafeBC-Common-Engineering/Common.Engineering.Discussion#7] [Technical Blog]
# fail-learn-reset

https://wsbctechnicalblog.github.io/fail-learn-reset.html

**Comment by AdamAndrews64** [comment]
Albert Einstein said  “Failure is success in progress”. Whether it's Elon Musk or Mahatma Gandhi, they all know that failure is a stepping stone of great things to come.
